### PR TITLE
[ticket/13879] Replaced px with em where appropriate.

### PR DESCRIPTION
**Pull request by callumacrae**
We're using ems basically everywhere else so ¯\\\_(ツ)_/¯

This will make it much easier for people to change the font-size of their board. I can now double the size of all the text in my board by just increasing the font-size of the body.

https://tracker.phpbb.com/browse/PHPBB3-13879

### CODE REVIEW

**Comment by Louis7777** [comment]
I was wondering. How about using rems (root ems) instead of ems? Would there be any downsides?

**Comment by callumacrae** [comment]
Yeah, not supported in IE8. If we were running the CSS through a built tool it would be fine because we could just get it to turn them into px, but we're not.

**Comment by callumacrae** [comment]
Anyone going to merge this, or are we going to let it get merge conflicts and close it in six months? :smile:

File: phpBB/styles/prosilver/theme/buttons.css [hunk 1]
```diff
@@ -5,8 +5,7 @@
 	cursor: pointer;
 	display: inline-block;
 	height: 18px;
-	line-height: 18px;
```
Comment (marc1706) [inline phpBB/styles/prosilver/theme/buttons.css]: Is line-height not needed due to the height parameter?

Comment (callumacrae) [inline phpBB/styles/prosilver/theme/buttons.css]: It's not needed due to the `font-size`, iirc.

**Comment by RMcGirr83** [comment]
@marc1706 

http://area51.phpbb.com/phpBB/viewtopic.php?f=126&t=48186

This broke the responsive layout and needs to be fixed/reverted

**Comment by callumacrae** [comment]
Fixed, not reverted, obviously.

**Comment by Nicofuma** [comment]
https://github.com/phpbb/phpbb/pull/3772

**Pull request merged**